### DISCUSSION TITLE
Audiogridder - Connection server/master pc

### DISCUSSION THREAD

**Discussion by Lele331** [apohl79/audiogridder#462] [Help]
Hello everyone.
I have a problem with using Gredder. I am trying to connect from Cubase of the master pc to the slave server. In fact, the master machine can see the address of the slave server on the network. As you can see from the photo it also makes me click on "connect" but in fact in the general window of the plugin it is not connected and in fact it does not make me choose any vst.
Can you help me?
Thanks!
![Screenshot (928)](https://user-images.githubusercontent.com/81808453/113410526-eb3cee00-93b3-11eb-9007-99eb7e1dc7f5.png)

**Comment by CaBuffalo** [comment]
Go to your slave server and restart AG. See if it works.

On Fri, Apr 2, 2021 at 4:04 AM Lele331 ***@***.***> wrote:

> Hello everyone.
> I have a problem with using Gredder. I am trying to connect from Cubase of
> the master pc to the slave server. In fact, the master machine can see the
> address of the slave server on the network. As you can see from the photo
> it also makes me click on "connect" but in fact in the general window of
> the plugin it is not connected and in fact it does not make me choose any
> vst.
> Can you help me?
> Thanks!
> [image: Screenshot (928)]
> <https://user-images.githubusercontent.com/81808453/113410526-eb3cee00-93b3-11eb-9007-99eb7e1dc7f5.png>
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/apohl79/audiogridder/discussions/462>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ARMAR6ED4JINWM3E3YE5VOTTGWQFHANCNFSM42IWPCMA>
> .
>


-- 
Kevin Ma
Supported by Gmail.

**Reply by Lele331** [reply]
I always try to restart but it doesn't work

**Reply by Clintcinnati** [reply]
Try restarting the server machine and make sure wifi is off on that machine, then once booted and the server is running, Close the daw session and re- open that so the AG plugin(s) try to re-connect to the hopefully healed server(s)

**Reply by Lele331** [reply]
thanks for all suggestions but it doesn't work

**Comment by CaBuffalo** [comment]
For better performance, wifi is not recommended. It causes CPU spikes
because of the slow speed.

Please consider LAN connection.

On Sun, Apr 4, 2021 at 12:37 PM Clintcinnati ***@***.***>
wrote:

> Try restarting the server machine and make sure wifi is off on that
> machine, then once booted and the server is running, Close the daw session
> and re- open that so the AG plugin(s) try to re-connect to the hopefully
> healed server(s)
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/apohl79/audiogridder/discussions/462#discussioncomment-568512>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ARMAR6C4N7ZDCMWTHH3PX5LTHC5Y7ANCNFSM42IWPCMA>
> .
>


-- 
Kevin Ma
Supported by Gmail.

**Reply by Lele331** [reply]
I had already deactivated the wifi of both the master and the slave. I dedicated a small LAN network to them. In fact with VEP I have no problems using it

**Comment by CaBuffalo** [comment]
Did you check the firewall setting in your server? AG asks for permission
to connect when you install it, if I'm correct.

On Mon, Apr 5, 2021 at 3:10 AM Lele331 ***@***.***> wrote:

> I had already deactivated the wifi of both the master and the slave. I
> dedicated a small LAN network to them. In fact with VEP I have no problems
> using it
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/apohl79/audiogridder/discussions/462#discussioncomment-569887>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ARMAR6F5NLELBFACW6ZEKS3THGD7RANCNFSM42IWPCMA>
> .
>


-- 
Kevin Ma
Supported by Gmail.

**Reply by Lele331** [reply]
In the firewire settings the authorization to AG has been given but it still doesn't work. Nobody can show me other solutions? It is a shame because using AG would be a turning point

**Comment by apohl79** [comment]
I assume you are having an issue with the server not being able to connect the workstation computer. Can you ping the workstation from the server?

Also try turning off the firewall completely.

Version 1.2 will remove the back connection thing, so it should work through gateways etc as well.

**Reply by Lele331** [reply]
Thanks @apohl79. I tried but it doesn't work. Consider that I already use VEP (master / slave pc) without problem.

**Reply by apohl79** [reply]
The problem is very likely the "back connection" from the server to the plugin. In version 1.2 the client will be fully passive, which should fix your issue. The first beta will be available soonish.

**Reply by Lele331** [reply]
I will wait for the new version

**Comment by Iceaddiction** [comment]
Hi guys, I have the same issue here, new version, windows 11. The vst finds the server but it's never connected. The server has a fixed ip and the firewall down...any help?
